pkg/localize/locales/en/cmd/kafka_update.en.toml

```diff
@@ -36,30 +39,49 @@ one = 'Name of the Kafka instance you want to update'
 description = 'Description for the --owner flag'
 one = 'ID of the user you want to set as the owner of this Kafka instance'
 
+[kafka.update.flag.reauthentication]
+one = 'Enable connection reauthentication for the Kafka instance'
+
 [kafka.update.flag.yes]
 one = 'Forcibly update the Kafka instance without confirmation'
 
 [kafka.update.summaryTitle]
 one = 'Summary of Proposed Changes'
 
+[kafka.update.reauthentication.disclaimer]
+one = '''
+If a bad actor obtains credentials or service account to Kafka instance that has disabled reauthentication,
+deactivating the user account or service account will not disconnect already established connections.
+
+Options to disconnect client when reauthentication is disabled:
+- Use ACLs to prevent unauthorized connections from performing any operation
```

Comment:
When `-` is used both markdown and ascidocs rendering doesn't provide values for that properly. 
We should use `*` across entire cli for lists. 

Note: let's create seprate issue